katalog/istio/minimal/README.md

```diff
@@ -0,0 +1,96 @@
+# Istio (Minimal)
+
+As described [before](../../istio/README.md#modular), this [Istio](../../istio) package has been designed to make it
```

Comment:
- If you change Modular to Modules in the other file remember to update the anchor of this link.
- Package or module?

Reply:
>     * If you change Modular to Modules in the other file remember to update the anchor of this link.

Sure!

>     * Package or module?

Let's say that `fury-kubernetes-service-mesh` is a Fury module and what you can find inside `katalog` directory are packages. We can discuss about `subpackages`. How do you name `citadel`: a `package` or a `subpackage`?